pypi_formula_mappings.json

```diff
@@ -304,6 +304,9 @@
   "git-review": {
     "exclude_packages": ["certifi"]
   },
+  "git-revise": {
+    "extra_packages": ["setuptools"]
+  },
```

Comment:
`setuptools` is build-only so this shouldn't be needed (our `venv` [enables build isolation](https://github.com/Homebrew/brew/blob/3802d113ae74fc428be5092d662e9eafc26e6b8b/Library/Homebrew/language/python.rb#L376) meaning `pip` will install it if needed):
```suggestion
```

Reply:
Thanks @branchvincent, I'll update the rest accordingly too.

Reply:
sounds good, there is an edge case where `pip` wont download setuptools if the project doesn't have a `pyproject.toml`, but we can workaround that with https://github.com/Homebrew/homebrew-core/pull/163143#discussion_r1493826573